pkg/controller/registry/resolver/cache.go

```diff
@@ -453,17 +450,6 @@ func WithPackage(pkg string) OperatorPredicate {
 	}
 }
 
-func WithoutDeprecatedProperty() OperatorPredicate {
-	return func(o *Operator) bool {
-		for _, p := range o.bundle.GetProperties() {
-			if p.GetType() == opregistry.DeprecatedType {
-				return false
-			}
-		}
-		return true
-	}
-}
-
```

Comment:
Why are we removing this code block? I might be missing something obvious, but I didn't see anything in the 4.8 PR for removing fields from this cache package. Just want to rule out whether this is an artifact of the cherry-pick failing and rebase being needed.

Reply:
Link to that block of code in the 4.8 PR: https://github.com/operator-framework/operator-lifecycle-manager/pull/2104/files#diff-5311e3f964b627fa6618423cf9e1ce3f7c84530d3045b93365f1102cea3280b8R464.

Reply:
deprecated properties are handled by the solver instead: https://github.com/operator-framework/operator-lifecycle-manager/pull/2313/files/40187e79db7ab1b5fb4b66bcecce198a03f8313f#diff-2c070869d0e7b6bf96c69fbdcde88e430d5d8781e047c1d96cb22ee0967c0111R64-R72